frontend/src/lib/services/Libre311/Libre311.ts

```diff
@@ -389,6 +394,40 @@ export type CreateServiceDefinitionAttributesParams = z.infer<
 	typeof CreateServiceDefinitionAttributesSchema
 >;
 
+export const EditServiceDefinitionAttributesSchema = z.object({
```

Comment:
I think this type is redundant and you can just use `ServiceDefinitionAttributeSchema`